spacy/strings.pyx

```diff
@@ -315,3 +315,25 @@ cdef class StringStore:
         self._map.set(key, value)
         self.keys.push_back(key)
         return value
+
+    @cython.boundscheck(False)  # Deactivate bounds checking
+    cdef const unsigned char* utf8_ptr(self, const attr_t hash_val):
+        if hash_val == 0:
+            return b""
```

Comment:
Thanks, should be fixed now. Btw because not all the representations within the string store are 0-terminated, I've now moved away from using 0-termination and use the declared lengths of each string instead.